nixio/block.py

```diff
@@ -91,15 +91,15 @@ def create_multi_tag(self, name="", type_="", positions=None,
         try:
             if not isinstance(positions, DataArray):
                 da_name = "{}-positions".format(name)
-                positions = self.create_data_array(da_name,
-                                                   "{}-positions".format(type_),
-                                                   data=positions)
+                positions = self.create_data_array(
+                    da_name, "{}-positions".format(type_), data=positions
+                )
```

Comment:
you change the indentation here, but keep it the old style a few lines later :)
I am actually fine with both ways :man_shrugging:

Reply:
It's a line length issue, not so much indentation style.

I mean
```python
Foo(one,
    two,
    three)
```

and

```python
Foo(
    one,
    two,
    three
)
```
are both "valid" (conventionally).  I prefer the former, but sometimes the line length makes the latter necessary.